src/utils/db/sqlAPI.ts

```
@@ -60,12 +60,74 @@
+      r128gain: r128GainTable.r128gain,
+    })
+    .from(r128GainTable)
+    .all();
+
+  const res: {
+    [id: string]: { itemid: string; abrepeat?: string; r128gain?: number };
+  } = {};
+
+  abrepeat.reduce(
+    (acc, curr) => ({
+      ...acc,
```

Biome (1.9.4)

[error] Avoid the use of spread (`...`) syntax on accumulators.

Spread syntax should be avoided on accumulators (like those in `.reduce`) because it causes a time complexity of `O(n^2)`.
Consider methods such as .splice or .push instead.

(lint/performance/noAccumulatingSpread)